src/helpers/github.rs

```diff
@@ -609,9 +740,17 @@ impl std::str::FromStr for GitHubRepoSourceKind {
             s if s.starts_with("repos/") && num_of_slashes == 2 => {
                 Ok(GitHubRepoSourceKind::Repo(s[6..].to_string()))
             }
+            s if s.starts_with("gists/") && num_of_slashes == 1 => {
+                let gist = s[6..].to_string();
```

Comment:
I believe Rust lets you do something amazing:

```rust
if let Some(gist) = s.strip_prefix("gists/") {
  ...
}
```

It'll only match if it has the prefix, and will give you everything which follows in a single call - I think it's a fair bit clearer than the magic numbers too 😁